File: interlayer/src/main/java/com/street/interlayer/service/WalletService.java
package com.street.interlayer.service;

import com.street.interlayer.model.Wallet;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public interface WalletService {

    Wallet getWalletById(String id);
    List<Wallet> getAllWallets();
    Wallet saveWallet(Wallet wallet);
    void deleteWallet(String id);

}

File: micro/src/main/kotlin/com/street/micro/service/WalletService.kt
package com.street.micro.service

import com.street.micro.entity.Wallet
import org.springframework.stereotype.Service

@Service
interface WalletService {

    fun getWalletById(id: String): Wallet
    fun getAllWallets(): List<Wallet>
    fun saveWallet(wallet: Wallet): Wallet
    fun deleteWallet(id: String)

}